### PR TITLE
Add possibility to select a row/tr at component (Data Table) load

### PR DESCRIPTION
**Pull request by vaceslav**
At the moment it is not possible to select a row at component load.

### CODE REVIEW

File: examples/components/data_table/selectable_usage.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id">
+  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id" [is-active]="material.id == 2">
```
Comment (Gregcop1) [inline examples/components/data_table/selectable_usage.html]: This shouldn't be a part of this example in my opinion. The doc is enough for me

File: ng2-material/components/data_table/data_table_selectable_tr.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ export abstract class AbstractMdDataTableSelectableRow implements AfterContentIn
   selectableValue: string;
   @Output()
   onChange: EventEmitter<ITableSelectableRowSelectionChange> = new EventEmitter(false);
+  @Input('is-active')
```
Comment (Gregcop1) [inline ng2-material/components/data_table/data_table_selectable_tr.ts]: Can you group `@Input` together and put the `@Ouput` below please?

**Comment by Gregcop1** [comment]
Good Job. Can you write some tests about using or not this `@Input` ?

**Comment by Gregcop1** [comment]
And at the end squash everything in one useful commit please

File: examples/components/data_table/selectable_usage.html (hunk 1 above)
Comment (ollwenjones) [inline examples/components/data_table/selectable_usage.html]: I don't mind it, just comparing the id to `2` isn't a really clearly useful example though.

File: ng2-material/components/data_table/README.md [hunk 3]
```diff
@@ -39,6 +39,7 @@ MdDataTable is an enhancment of classic data tables.
 | --- | --- | --- | --- |
 | md-data-table-selectable-row | tbody tr | boolean | Enable a checkbox for this row. |
 | selectable-value | tbody tr | string | value of the checkbox. If it's not set the checkbox's value will be the index of the row. |
+| is-active | tbody tr | boolean | initial value for  the checkbox. If true the row will be selected. |
```
Comment (ollwenjones) [inline ng2-material/components/data_table/README.md]: I know it's a bit late to complain, but the "is active" verbiage always bordered on confusing to me. We're really talking about whether a row/datum is _selected_, but I realize changing it now would constitute a breaking change. :disappointed: Same with select _able_ - its not the _able -- ability_ that is changing it's the select _ion_ state. I guess _selectable_ is a tie to the name of the component though.

**Comment by ollwenjones** [comment]
This seems simple and flexible enough. It's pretty common for complex tables to have a "selection model" and I do think it's pretty important we can assign the selections from the outside. This creates no opinions for what/where the selection model _should be_, but provides a hook for them to tie it into the row rendering. :+1:

**Comment by vaceslav** [comment]
Hello @All,
I am "new" to the git and opensource.
Is any changes required from me? Should I change  docs 
> This shouldn't be a part of this example in my opinion. The doc is enough for me

or group inputs?
> Can you group @Input together and put the @Ouput below please?

File: ng2-material/components/data_table/README.md (hunk 3 above)
Comment (Gregcop1) [inline ng2-material/components/data_table/README.md]: Let's change it in other PR and take in consideration a period of deprecation

**Comment by Gregcop1** [comment]
Hello. Yes, you have to treats all comments. To sum up:
- remove ` [is-active]="material.id == 2"` from `examples/components/data_table/selectable_usage.html`
- group @Input together in `components/data_table/data_table_selectable_tr.ts`
- writing tests about the new @Input
- squash everything in one commit

**Comment by vaceslav** [comment]
Hello,
I implemented the tasks from @Gregcop1 except squash to one commit.
I have developed in the master brach. Can anyone explain how can i do this?

Why failed the Travis buid?

**Comment by ollwenjones** [comment]
@vaceslav you need to review the [contributing guidelines](https://github.com/justindujardin/ng2-material/blob/master/CONTRIBUTING.md) for branch name & commit format, etc., then probably `git reset` to the last commit before your first commit (**not** hard - a soft reset will move the changes to your workspace, erasing the commit history) then you can branch. The changed files will stay 'dirty' in your workspace, and you can then commit one commit with the proper message format. This will acheive the same thing as "squashing" but will also remove the history changes to the master, and put it in your feature branch instead.

Wouldn't hurt to google around and double check me, or make an extra backup branch before the reset just in case.

In future it's best to branch before doing anything, so you have no chance of messing up `master` and can avoid some of this git work. It's as easy as `git checkout -b <branch-name>`

**Comment by vaceslav** [comment]
I created a new feature branch https://github.com/vaceslav/ng2-material/tree/feature/select-row-at-load
When I reset my master branch what happened with this pull request? and should I create a new pull request  for the feature branch?

Sorry for these stupid beginners questions, but this is my first pull request ever.

**Comment by ollwenjones** [comment]
@vaceslav no such thing as "stupid beginner quesitons" - only stupid beginners are beginners too proud to ask questions. :wink: 

Where I work, a master to master PR would have been rejected outright, so I didn't think it through correctly for the sake of _this pull request_. I honestly can't tell you if others will recommend a new PR or continuing this one. It might not be a big deal that you are going from master of fork?

If you/we want to preserve this pull request, you will need to squash the commits on your fork's master. [interactive rebase](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

**Comment by vaceslav** [comment]
So now it is on my master branch. With one commit and tests :-)

**Pull request closed without merging**